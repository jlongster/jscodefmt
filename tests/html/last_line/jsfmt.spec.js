run_spec(__dirname, ["html"], { jsxBracketSameLine: true });
run_spec(__dirname, ["html"], { jsxBracketSameLine: false });
