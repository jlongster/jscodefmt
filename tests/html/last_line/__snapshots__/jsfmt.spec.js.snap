// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last_line.html - {"jsxBracketSameLine":false} format 1`] = `
====================================options=====================================
jsxBracketSameLine: false
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<article
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars">...</article>
<span
id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars">...</span>

=====================================output=====================================
<article
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars"
>
  ...
</article>
<span
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars"
  >...</span
>

================================================================================
`;

exports[`last_line.html - {"jsxBracketSameLine":true} format 1`] = `
====================================options=====================================
jsxBracketSameLine: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<article
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars">...</article>
<span
id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars">...</span>

=====================================output=====================================
<article
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars">
  ...
</article>
<span
  id="electriccars"
  data-columns="3"
  data-index-number="12314"
  data-parent="cars"
  >...</span
>

================================================================================
`;
